src/spikeinterface/sorters/runsorter.py

```diff
@@ -96,23 +96,6 @@
         If True, the output Sorting is returned as a Sorting
     delete_container_files : bool, default: True
         If True, the container temporary files are deleted after the sorting is done
-    extra_requirements : list, default: None
-        List of extra requirements to install in the container
-    installation_mode : "auto" | "pypi" | "github" | "folder" | "dev" | "no-install", default: "auto"
-        How spikeinterface is installed in the container:
-          * "auto" : if host installation is a pip release then use "github" with tag
-                    if host installation is DEV_MODE=True then use "dev"
-          * "pypi" : use pypi with pip install spikeinterface
-          * "github" : use github with `pip install git+https`
-          * "folder" : mount a folder in container and install from this one.
-                      So the version in the container is a different spikeinterface version from host, useful for
-                      cross checks
-          * "dev" : same as "folder", but the folder is the spikeinterface.__file__ to ensure same version as host
-          * "no-install" : do not install spikeinterface in the container because it is already installed
-    spikeinterface_version : str, default: None
-        The spikeinterface version to install in the container. If None, the current version is used
-    spikeinterface_folder_source : Path or None, default: None
-        In case of installation_mode="folder", the spikeinterface folder source to use to install in the container
```

Comment:
Why delete this?

Reply:
This is a list of kwargs which are common between `run_sorter` and `run_sorter_container`. The deleted ones only apply for `run_sorter_container` can be found in its docstring.